docker-compose.yml

```diff
@@ -17,3 +17,4 @@ web:
     - db
   environment:
     - PYTHONUNBUFFERED=1
+    - SERVER_NAME=docker.local:5000
```

Comment:
This exact name isn't required, but something like it it. This is due to how cookies get set by browsers. Essentially, we need a server name set and a `.` in that name. See the "More on `SERVER_NAME`" section of Flask's [configuration docs](http://flask.pocoo.org/docs/0.10/config/) for more details.

I'm open to suggestions here.